tools/e2e/Taskfile.yaml

```diff
@@ -28,10 +28,9 @@ tasks:
       DATALENS_UI_TAG: "{{.UI_TAG}}"
 
   datalens-ui-repo-clone:
-    dir: "{{.DATALENS_UI_FOLDER}}/.."
-    desc: Clone datalens-ui (repository) near primary repository
+    desc: Clone datalens-ui (repository) to a specified location
     cmds:
-      - git clone git@github.com:datalens-tech/datalens-ui.git
+      - if [ ! -d {{.DATALENS_UI_FOLDER}} ]; then git clone git@github.com:datalens-tech/datalens-ui.git {{.DATALENS_UI_FOLDER}}; fi
```

Comment:
https://taskfile.dev/usage/#by-fingerprinting-locally-generated-files-and-their-sources

conditinal tasks can be used probably with `generates`

Reply:
I tried it but with no luck so far, because we don't have anything to specify as `sources`

Reply:
I think @MCPN used Taskfile itself as source in such a case somewhere :D